index.html

```diff
@@ -97,13 +97,13 @@
   <body>
     <section id='abstract'>
       <p>
-        The <cite>Push API</cite> provides <a title="webapp">webapps</a> with scripted access to
-        server-sent messages, for simplicity referred to here as <a title="push message">push
-        messages</a>, as delivered by <a title="push service">push services</a>. A <a>push
-        service</a> allows an <a>application server</a> to send messages to a <a>webapp</a>,
-        regardless of whether the <a>webapp</a> is currently active on the <a>user agent</a>. The
-        <a>push message</a> will be delivered to a <a>Service Worker</a>, which could then store
-        the message's data or display a notification to the user.
+        The <cite>Push API</cite> enables sending of a <a>push message</a> to a <a>webapp</a> via a
+        <a>push service</a>. An <a>application server</a> can send a <a>push message</a> at any
+        time, even when the <a>webapp</a> is active or the <a>user agent</a> is inactive. The
```

Comment:
This is the more important question: "even when the webapp is active" sounds strange, certainly things usually work when they are active, it requires no special mention. Did you mean 'inactive'?